lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -77,7 +77,14 @@ Function Chocolatey-Install-Upgrade
             Fail-Json $result "Chocolatey bootstrap installation failed."
         }
         $result.changed = $true
-        $script:executable = "C:\ProgramData\chocolatey\bin\choco.exe"
+        if (Test-Path "Env:\ChocolateyInstall")
+        {
+            $script:executable = "{0}\bin\choco.exe" -f (Get-ChildItem "Env:\ChocolateyInstall").Value
```

Comment:
Somehow I prefer to just concatenate both strings, not sure what real PowerShell people in the Windows Working Group prefer.